app/workers/moderator/sink_articles_worker.rb

```diff
@@ -0,0 +1,20 @@
+module Moderator
+  class SinkArticlesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(user_id)
+      user = User.find_by(id: user_id)
+      return unless user
+
+      articles = Article.where(user: user)
+      reactions = Reaction.where(reactable: articles)
+      new_score = reactions.sum(:points) + Reaction.where(reactable: user).sum(:points)
+      articles.update_all(score: new_score)
+    rescue StandardError => e
+      DataDogStatsClient.count("moderators.sink", 1, tags: ["action:failed", "user_id:#{user.id}"])
+      Rails.logger.error("Error while sinking user: #{e}")
```

Comment:
Rather than logging this, should we send it to Honeybadger instead? My thought is that some moderators probably dont have access to our logs and Datadog, however, our Honeybadger dashboard is public so if something broke in this process they could go there to find out what/why